ingest/mappings/mineral-observation.json

```diff
@@ -148,6 +142,25 @@
             "type":"string"
           }
         }
+      },
+      "mineral-specimen":{
```

Comment:
Let's make the description of the age determination process an object with a label and description.

example:
```
"mineral-specimen" : {
  "age-determination-process": {
    "label": "G",
    "description": "This mineral is directly dated." 
  }
}
```